templates/serverpod_templates/projectname_server_upgrade/Dockerfile

```diff
@@ -3,7 +3,7 @@
 FROM dart:3.2.5 AS build
 
 WORKDIR /app
-COPY . .
```

Comment:
Question: I don't understand the motivation behind this change.

I can successfully build the image in the `build` container when testing.

The above `WORKDIR` command sets the path inside of the build container to `/app`. So `COPY . .` __should__ provide the same result as `COPY . /app`.

Reply:
Note that we are not copying the full project from the build container into the final container that runs the server. We only copy the files/folders listed on line 20-24.

Reply:
This works on Linux but not with Docker for Mac

Reply:
This seems to have been an issue with docker desktop for mac, but I cannot reproduce it anymore, so Ill revert it :)